k8s/app.env.yaml.j2

```diff
@@ -89,4 +103,4 @@
               valueFrom:
                 secretKeyRef:
                   name: dev-portal-secrets
-                  key: OIDC_RP_CLIENT_SECRET
+                  key: OIDC_RP_CLIENT_SECRET
```

Comment:
Is the missing line of whitespace gonna make a linter choke somewhere?

Reply:
Yup this should be fine, apologies my client actually wipes out whitespaces after exit